api/wasm/indigo-ketcher/indigo-ketcher.cpp

```diff
@@ -320,6 +321,61 @@ namespace indigo
         iko.set(mid, IndigoKetcherObject::EKETMolecule);
     }
 
+    class VectorString
```

Comment:
Since there is `VectorStringBracketed`, it makes sense to name this one as `VectorStringSemicolonSeparated` or laconic `VectorStringSemicoloned` :)